File: C/Assignments/Assignment 6/Assignment 4 with Function/q3.c
/*3. Print perfect numbers in the given range 1 to n.
*/

void printperfect(int n){
	for(int i=1 ; i <= n ; i++) {
		int no = i ;
		int sum = 0 ;
	
		for(int j = 1 ; j <= no / 2 ; j++){
			if(no % j == 0) {
				sum = sum + j ;
			}
		}
		
		if(sum == no){
			printf("%d   ", no);
		}
	}
}


void main() {
	
	int n ;
	printf("enter number : ");
	scanf("%d" , &n);
	
	printperfect(n);
	
	
}

File: C to Java/Assignments/Assignment 3/q3.java
/* Sum of numbers in given range.
Find sum of numbers from start to end.
Input: start = 1, end = 5
Output: 15  */


import java.util.Scanner;

public class q3 {

    public static void main(String args[]) {

        int start = 1 , end = 5 , sum = 0;
	while(start <= end) {
		sum = sum + start ;
		start++ ;
	}
	
	System.out.printf("sum is : %d", sum);

	
	
        }
}